index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f0e9415ee2.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>William Raga</title>
+  </head>
+  <body>
+    <header>
+      <a href="" class="white text-normal align-center">William Raga</a>
+      <nav>
+        <i class="fa-solid fa-bars fa-2x white"></i>
+      </nav>
+    </header>
```

Comment:
![portfolio-1](https://user-images.githubusercontent.com/49184579/176661004-9a83d785-2429-4d8f-a02c-ee0ae51cf3a2.png)

Please have a look at the above comparison of your project and the Figma Design. It is imperative that you follow the Figma Design.

1. There is no top margin of the navigation bar
2. The font should be roboto
3. Margin is quite large compared to Figma
4. Margin/padding issue
5. Margin/padding issue
6. Color issue
7. Margin/padding issue
8. Margin/padding issue
9. Size of the icons are small
10. Margin/padding issue

Reply:
I'm I supposed to fill my own details in the project or use the content as in the figma?

Reply:
@itsmraga-hub The content is not the issue. Figma Design is about styling in your css. The ten points I pointed out are all about styling.